src/components/Teams/TeamGoalsList.vue

```diff
@@ -63,7 +66,10 @@ export default {
 		query: teamsGoals,
 		preFetch: true,
 		result(result) {
-			this.activeGoals = result?.data?.getGoals?.values ?? [];
+			const teams = _groupBy(result?.data?.getGoals?.values ?? [], 'teamId');
+			this.activeGoals = _map(teams, goals => {
+				return _orderBy(goals, 'startDate', 'desc')[0];
```

Comment:
Here `startDate` is a string. Will sorting work consistently? I'm assuming the string might sort like expected, but you might need to convert the date strings to actual date objects.